src/python_pybind11/CMakeLists.txt

```diff
@@ -1,10 +1,3 @@
-if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
Sorry, I see the comment about this in the description: 

>  The Win32 PYTHON_LIBRARIES (PYTHON_DEBUG_LIBRARIES) variable was also removed as there is no equivalent in the new CMake module for getting the debug libraries

Could we please keep the code for the versions of CMake that were using it before. Otherwise we could be opening the door to regressions.

Reply:
Done.